test/test_ops.py

```diff
@@ -22,6 +22,14 @@
 # Get names of all the operators which have entry in `method_tests` (legacy testing infra)
 method_tested_operators = set(map(lambda test_details: test_details[0], method_tests()))
 
+# Assure no operator has `dtypes=_DYNAMIC_DTYPES`
+def is_dynamic_dtype_set(op):
+    dyn_dtype = set(_DYNAMIC_DTYPES)
+    return (op.dtypes == dyn_dtype) or (op.dtypesIfCPU == dyn_dtype) or (op.dtypesIfCUDA == dyn_dtype)
+
+filtered_ops = list(filter(is_dynamic_dtype_set, op_db))
+assert len(filtered_ops) == 0, "One of the operator has dtypes=`DYNAMIC_DTYPES`"
```

Comment:
Added an assert here.
This was much faster than adding a test for each operator to verify the same.

We can pin-point the offending operator here as well.

Reply:
Cool idea; maybe the right pattern for the logic in `test_duplicate_method_tests`, too?

When will this assert trigger? Will the other tests run first so people can see if they have other failures?

Reply:
An additional idea is to use 

https://github.com/pytorch/pytorch/blob/fa0b89bbf7eb8e106930accc5324f47b2a76370f/torch/testing/_internal/common_utils.py#L238

and only warn if running locally but error (after the tests have run) in PyTorch CI